pysteps/visualization/motionfields.py

```diff
@@ -238,11 +209,9 @@ def streamplot(
     map=None,
     geodata=None,
     drawlonlatlines=False,
-    basemap_resolution="l",
-    cartopy_scale="50m",
     lw=0.5,

```

Comment:
**lw** keyword is not used in streamplot. Perhaps it should be added to the plot_geography call in streamplot? (like in quiver).

Reply:
well spotted! corrected in aaee5d1